docs/source/getting-started/model-registry/step3-model-version-alias.rst

```diff
@@ -0,0 +1,82 @@
+Alias Model Version
+===================
+
+Model version aliases are available for MLflow >= `2.3.0`.
+
+Model version aliases provide a flexible way to set named aliases on model versions. For example, 
+setting a champion alias on a model version enables you to fetch this model version by that alias 
+via client API :py:func:`mlflow.client.MlflowClient.get_model_version_by_alias()` or the model URI 
+`models:/<registered model name>@champion`. Aliases can be easily reassigned to new model versions 
+via the UI and client API alike, thereby decoupling model deployment from the production system 
+code. Unlike model registry stages, more than one alias can be applied to any given model version, 
+creating powerful possibilities for model deployment.
+
+To learn more, visit `here <https://mlflow.org/docs/latest/model-registry.html#deploy-and-organize-models-with-aliases-and-tags>`_.
+
+
+Step 1: Get the Model with the MLflow Client
```

Comment:
I actually think it would easier to alias the model version via the UI, rather than the MLflow Clientk, see more in my larger PR review comment.

Reply:
IMO we should keep the programmatic solution because:

- MLflow is typically a code-first tool. Most customers interact with models via client and not the UI.
- It's easier for the user to replicate on their system via copying code versus duplicating UI steps.

Reply:
Does this make sense?